modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/jaxrs/exception/mapper/RequiredObjectFieldRelationshipExceptionMapper.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.admin.rest.internal.jaxrs.exception.mapper;
+
+import com.liferay.object.exception.RequiredObjectFieldRelationshipException;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.vulcan.accept.language.AcceptLanguage;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Larissa Ribeiro
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Object.Admin.REST)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Object.Admin.REST.RequiredObjectFieldRelationshipExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class RequiredObjectFieldRelationshipExceptionMapper
+	extends BaseExceptionMapper<RequiredObjectFieldRelationshipException> {
+
+	@Override
+	protected Problem getProblem(
+		RequiredObjectFieldRelationshipException
+			requiredObjectFieldRelationshipException) {
+
+		return new Problem(
+			Response.Status.BAD_REQUEST,
+			_language.format(
+				_acceptLanguage.getPreferredLocale(),
+				requiredObjectFieldRelationshipException.getMessageKey(),
+				requiredObjectFieldRelationshipException.getArguments()));
```

Comment:
Could you use `ObjectExceptionMapperUtil.getTitle` instead `_language.format`, please? Something like this:

```java
ObjectExceptionMapperUtil.getTitle(
						_acceptLanguage,
						requiredObjectFieldRelationshipException.getArguments(), _language,
						requiredObjectFieldRelationshipException.getMessage(),
						requiredObjectFieldRelationshipException.getMessageKey())
```

Reply:
Sure, @murilostodolni, just did it!